proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -70,6 +74,20 @@ service RepositoryService {
   rpc DeleteRepository(DeleteRepositoryRequest) returns (DeleteRepositoryResponse);
   // DeleteRepositoryByFullName deletes a repository by full name.
   rpc DeleteRepositoryByFullName(DeleteRepositoryByFullNameRequest) returns (DeleteRepositoryByFullNameResponse);
+  // DeprecateRepositoryByName deprecates the repository.
+  rpc DeprecateRepositoryByName(DeprecateRepositoryByNameRequest) returns(DeprecateRepositoryByNameResponse);
+  // UndeprecateRepositoryByName makes the repository not deprecated and removes any deprecation_message.
+  rpc UndeprecateRepositoryByName(UndeprecateRepositoryByNameRequest) returns(UndeprecateRepositoryByNameResponse);
+  // GetRepositoriesByFullName gets repositories by full name.
+  rpc GetRepositoriesByFullName(GetRepositoriesByFullNameRequest) returns (GetRepositoriesByFullNameResponse);
```

Comment:
tiny bikeshedding thing: I would name this `GetRepositoriesByFullNames` personally, but I could go either way

Reply:
I considered both and had a slight preference for the singular version because we are getting each one by its full name.